### ISSUE TITLE
Credentials don't work anymore

### ISSUE THREAD

**Issue by junming** [agonzalezro/python-opensubtitles#11]
As of last week the credentials doctest/doctest/en/OSTestUserAgent don't work anymore. I reckon this is deliberate from the guys at OpenSubtitles who want to remain in control and see who does what. Probably some filtering for the User Agent.
They recommend developers not to include credentials in this fashion: https://www.npmjs.com/package/opensubtitles-api
I know this is a bummer for a wrapper but unfortunately that's how things are.

**Comment by agonzalezro** [comment]
Hi @junming, thanks for the feedback.

I am not maintaining this library anymore, but it allows login (not just anonymouse usage), isn't that working either?

Sadly I will not be able to create a PR for this feature, but it you manage to make it work I would be glad to merge it.

Thanks again!

**Comment by junming** [comment]
It's not working because the problem is with the login, the User Agent parts seems to be checked in a way it wasn't before (I tested directly without the wrapper with the credentials you use and they don't work anymore).
I don't think this can be really be fixed unless you allow people to use their own credentials. If you provide new credentials you register, they will be blacklisted soon or later because more they will be considered as publicly available.

**Comment by agonzalezro** [comment]
Hi @junming, thanks for the clarification.

Could you manually change the `UserAgent` here https://github.com/agonzalezro/python-opensubtitles/blob/master/pythonopensubtitles/settings.py#L3 and let me know if it works?

Clearly, this should be more flexible and we should be able to at least send our own `Settings` class around (which doesn't seem optimal either).

**Comment by itaybia** [comment]
please use "OSTestUserAgentTemp" instead, as stated in:
https://trac.opensubtitles.org/projects%3Cscript%20type=/opensubtitles/wiki/DevReadFirst
under: How to request a new user agent

**Comment by junming** [comment]
@itaybia I know, I contacted them and that's why they changed it. ;)
BUT this is very important to keep in mind that this will be changed often and that will break the script. The solution is to create one's own account on Opensubtitles, and request the account to be changed (manually) to developer, as explained in your link.

@agonzalezro Sorry - I implemented my own solution in the end, and stopped using the package because there was actually very little code from it I used: login and hash. But I tested with my own solution and this new user agent, and my own user agent, and it doesn't seem to raise any problems.
